static/js/popovers.js

```diff
@@ -209,9 +209,9 @@ exports.toggle_actions_popover = function (element, id) {
             narrowed: narrow.active(),
         };
 
-        var ypos = elt.offset().top - message_viewport.scrollTop();
+        var ypos = elt.offset().top;
```

Comment:
@timabbott As I said in [this comment](https://github.com/zulip/zulip/issues/3741#issuecomment-281984557), `elt.offset().top` gave me the offset from the top of the `viewport` and **not** the top of the feed/`document` (at least in my tests in both Chrome and Firefox). 

According to jQuery docs the latter should occur. I haven't been able to source any documentation for this spooky behaviour. :disappointed: